web/app/js/components/TapEventTable.jsx

```diff
@@ -126,6 +126,65 @@ const formatTapLatency = str => {
   return formatLatencySec(str.replace("s", ""));
 };
 
+const headersStyles = {
```

Comment:
I think the UI and display looks great. In reviewing the whole `TapEventTable` component, I am concerned that the component is getting to be too long and complex. Would you be able to break the headers content into its own component file and call it from the `TapEventTable` component?

Reply:
Done 👍